jaxopt/_src/armijo_sgd.py

```diff
@@ -281,12 +287,16 @@ def update(self, params, state, *args, **kwargs) -> base.OptStep:
     stepsize = self.reset_stepsize(state.stepsize)
 
     (f_cur, aux), grad = self._value_and_grad_with_aux(params, *args, **kwargs)
+    new_num_fun_eval = state.num_fun_eval + 1
+    new_num_grad_eval = state.num_grad_eval + 1
 
     goldstein = self.reset_option == 'goldstein'
     stepsize, next_params, f_next = self._armijo_line_search(
       goldstein, self.maxls, params, f_cur, stepsize, grad, self._coef,
       self.decrease_factor, self.increase_factor, self.max_stepsize, args,
       kwargs)
+    # FIXME: update new_num_fun_eval, new_num_grad_eval, new_num_line_search_iter
+    # with outputs from armijo line search
```

Comment:
Does that mean that the current returned numbers are incorrect?

Reply:
Yes, this is a problem.

I will remove these values for the returned state, and add an issue specifying which solvers need to be updated to feature these values.